backend/test/resolver.spec.js

```diff
@@ -1,91 +1,220 @@
+            expect(data.errors).toBeUndefined();
+            expect(data.data.todos.length).toBe(1);
+            expect(data.data.todos[0].user.name).toBe(testUserName)
+        })
+    });
+
+
     describe('Mutations', () => {
         const exampleText = "example text";
-        it("creates entry", () => {
-            mutate({ mutation: CREATE_ENTRY, variables: { text: exampleText } })
-                .then((data) => {
-                    expect(data.data.createEntry.id).toBe(123987123981723)
-                    expect(data.data.createEntry.id).toBeGreaterThanOrEqual(0)
-                    expect(typeof data.data.createEntry.id).toBe('string')
-                });
+        const exampleText2 = "new example text";
+        it("creates todo", async () => {
+            const data = await mutate({mutation: CREATE_TODO, variables: {text: exampleText}});
+            expect(data.data.createTodo.text).toBe(exampleText);
+            expect(data.data.createTodo.user.name).toBe(testUserName)
+        });
+        it("creates and then updates todo", async () => {
+            const createData = await mutate({mutation: CREATE_TODO, variables: {text: exampleText}});
+            expect(createData.errors).toBeUndefined();
+            const todoId = createData.data.createTodo.id;
+            expect(createData.data.createTodo.text).toBe(exampleText);
+            expect(createData.data.createTodo.user.name).toBe(testUserName);
+            const updateData = await mutate({mutation: UPDATE_TODO, variables: {id: todoId, text: exampleText2}});
+            expect(updateData.errors).toBeUndefined();
+            expect(updateData.data.updateTodo.text).toBe(exampleText2);
         });
-        it("deletes entry", () => {
-            mutate({ mutation: DELETE_ENTRY, variables: { id: "3" } })
-                .then((data) => {
-                    expect(data.data.deleteEntry).toBe(true)
-                });
+        it("deletes todo", async () => {
+            const data = await mutate({mutation: DELETE_TODO, variables: {id: "3"}});
+            expect(data.errors).toBeUndefined();
+            expect(data.data.deleteTodo).toBe(true)
+        });
+
+        it("displays entries in alphabetically descending order", async () => {
+            const createTopEntry = await mutate({mutation: CREATE_TODO, variables: {text: "z test"}});
+            expect(createTopEntry.errors).toBeUndefined();
+            const todoListData = await query({query: ALL_TODOS_QUERY, variables: {page: 0}});
+            expect(todoListData.errors).toBeUndefined();
+            expect(todoListData.data.todos.length).toBeGreaterThan(0);
+            expect(todoListData.data.todos[0].text).toBe("z test");
+            const id = todoListData.data.todos[0].id;
+            const deleteTopEntrys = await mutate({mutation: DELETE_TODO, variables: {id: id}});
+            expect(deleteTopEntrys.errors).toBeUndefined();
+            expect(deleteTopEntrys.data.deleteTodo).toBe(true);
```

Comment:
It would be much better to see the actual array of todos with names. The reader can see if they're ordered alphabetically!